styles/desktop-about.css

```diff
@@ -0,0 +1,183 @@
+/* eslint-disable */
+@import url('https://fonts.googleapis.com/css2?family=Lato:wght@400;900&display=swap');
+
+head {
+  background-color: rgb(58, 58, 72);
+}
+
+.nav-logo {
```

Comment:
- [ ] When I tried to scroll down I noticed that the elements come over the top dark navigation bar which makes the user experience bad. Please try to use the appropriate z-index or any element that can help to fix that issue and give a better user experience:+1: 
https://www.loom.com/share/d819dd7938a5461ca2c110b431fb8faf